src/hardware/serialport/softmodem.cpp

```diff
@@ -23,12 +23,84 @@
 #include <stdlib.h>
 #include <string.h>
 #include <utility>
+#include <fstream>
+#include <sstream>
 
 #include "support.h"
 #include "serialport.h"
 #include "softmodem.h"
 #include "misc_util.h"
 
+static const char phoneValidChars[] = "01234567890*=,;#+>";
+
+class CPhonebookEntry {
```

Comment:
I  request here: I really dislike Hungarian notation, can we not introduce new classes using it? Just name it `PhonebookEntry`, please.

I know DOSBox code is inconsistent in this aspect, but Hungarian notation is used only in few places throughout the code.